src/Servers/Kestrel/test/Interop.FunctionalTests/Http3/Http3TlsTests.cs

```diff
@@ -0,0 +1,126 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Net;
+using System.Net.Http;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Connections;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Server.Kestrel.Core;
+using Microsoft.AspNetCore.Server.Kestrel.FunctionalTests;
+using Microsoft.AspNetCore.Testing;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Xunit;
+
+namespace Interop.FunctionalTests.Http3
+{
+    [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/35070")]
+    public class Http3TlsTests : LoggedTest
+    {
+        [ConditionalFact]
+        [MsQuicSupported]
+        public async Task ServerCertificateSelector_Invoked()
```

Comment:
I liked to keep Arrange/Act/Assert pattern going with HTTP/3 tests in this project. Could you add those comments?

Reply:
Resolved offline.